ee/codegen/src/__test__/project.test.ts

```diff
@@ -22,36 +22,30 @@ import { WorkflowProjectGenerator } from "src/project";
 describe("WorkflowProjectGenerator", () => {
   let tempDir: string;
 
-  function expectProjectFileToMatchSnapshot(
-    project: WorkflowProjectGenerator,
```

Comment:
It's dangerous for implementation code to be used in test fixtures bc it allows the same bug in implementation to also be true in tests